llvm/lib/Transforms/Utils/Evaluator.cpp

```diff
@@ -145,9 +160,18 @@ static bool isSimpleEnoughPointerToCommit(Constant *C) {
         isa<GlobalVariable>(CE->getOperand(0)) &&
         cast<GEPOperator>(CE)->isInBounds()) {
       GlobalVariable *GV = cast<GlobalVariable>(CE->getOperand(0));
+      bool Pruning;
```

Comment:
Should this be `Pruned` (past tense) since the pruning has already happened?

Reply:
Fixed, thanks.